CHANGELOG.md

```diff
@@ -239,6 +239,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Removed `HorovodPlugin.start_{training,evaluating,predicting}` hooks ([#10989](https://github.com/PyTorchLightning/pytorch-lightning/pull/10989))
 
 
+- Removed `Trainer._call_accelerator_hook` ([#10999](https://github.com/PyTorchLightning/pytorch-lightning/pull/10999))
```

Comment:
`_call_accelerator_hook` doesn't need to be mentioned. this was added in between minor versions and was never part of an official release. 

the changelog should mention that `Accelerator.on_train_start` was removed

Reply:
oh, makes sense. fixed